toolchain/lower/file_context.h

```diff
@@ -56,10 +56,17 @@ class FileContext {
 
   // Returns a lowered type for the given type_id.
   auto GetType(SemIR::TypeId type_id) -> llvm::Type* {
+    return GetTypes(type_id).first;
+  }
+
+  // Returns both the lowered llvm IR type and the lowered llvm IR debug info
+  // type for the given type_id.
+  auto GetTypes(SemIR::TypeId type_id) const
```

Comment:
I think this is something like this?
```suggestion
  auto GetTypeAndDebug(SemIR::TypeId type_id) const
```